lib/engines/content_object_store/test/factories/content_block_schema.rb

```diff
@@ -0,0 +1,10 @@
+FactoryBot.define do
+  factory :content_block_schema, class: "ContentObjectStore::ContentBlockSchema" do
+    block_type { "email_address" }
```

Comment:
We could go further while we're here to set these as transient attributes here and in any other factories as it should be easy to do while we're on the subject?

Reply:
Done! 👍